common/utils/src/main/resources/error/error-states.json

```diff
@@ -2933,6 +2933,12 @@
         "standard": "Y",
         "usedBy": ["SQL/Foundation", "PostgreSQL", "Redshift", "Oracle", "SQL Server"]
     },
+    "42001": {
+        "description": "Invalid encoder error",
+        "origin": "SQL/Foundation",
+        "standard": "Y",
+        "usedBy": ["SQL/Foundation"]
+    },
```

Comment:
@MaxGekk I am not sure how I should approach to naming new sqlstate, do you have some more info on what these fields are about?